APE_specutil.rst

```diff
@@ -0,0 +1,481 @@
+  there is room for a few other visualization packages in the ecosystem. It
+  will, however, have a plugin architecture that may allow users/developers to
+  add to it.
+
+In addition to the packages described above, there are a wide range of more
+domain-specific, experimental, or taste-specific packages that are likely to be
+desired in the field of astronomical spectroscopy. These packages can be
+developed as independent affiliated packages, and this APE encourages such
+development.  However, the intent is for such packages to not *duplicate*
+behavior in the foundational three packages described above. Because of the
+wide range of scientific use cases for spectra, it is understood that some
+level of duplication between affiliated packages is expected.  E.g., multiple
+line-fitting packages may be reasonable because different science cases may be
+better-suited to different line-fitting approaches.
+
+Packages should aim for the same level of testing and documentation found in
+the `astropy` package, and should follow the standard Astropy affiliated
+package process to do this.  This includes, in particular, considering overlap
+with existing packages - existing code should be considered and merged when
+appropriate.  The Spectroscopic Coordinating Committee will assist the Astropy
+Coordinating Committee with this process.  Developers are encouraged to build
+or update existing tools to meet their needs rather than create new packages.
+
+The Spectrum1D class
+--------------------
+
+A spectroscopy package should provide a representation of the data, tools for
+reducing spectroscopic observations, analysis tools, and methods for
+interacting and visualization of the spectra. In this section, we describe some
+of the functionality that will be required for a common spectroscopic package.
+While equally important, we leave the specification of multi-dimensional
+spectral objects with different spectral axes to future work.  At the same
+time, we should consider how these tools will be compatible with
+multi-dimensional data in the future.
+
+
+Attributes of Spectrum1D
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this context, a spectrum describes how the flux changes as a function of the
+energy of the radiation from some object.  There are a number of other
+parameters that can be used to describe a spectrum, and a ``Spectrum1D`` object
+will have the following properties:
+
+* ``spectral_axis``
+* ``flux``
+* ``uncertainty``
+* ``wcs`` (more detail on this attribute in the section below)
+* ``meta`` (a dict-like or maybe None)
+
+The ``spectral_axis`` is always a quantity with the same shape as the last (or
+only) dimension of ``flux``. It also must have a spectral unit (i.e., length,
+frequency, energy, or velocity).  To ease use, the Spectrum1D object will have
+``energy``, ``wavelength``, and ``frequency`` properties that convert the
+`spectral_axis` to the appropriate units.  These are to be considered the
```

Comment:
Should we also mention that we're supporting converting the spectral axis to velocity space (since we have an example down below)?

Reply:
Yes, I think that is what Adam is suggesting and would like to see.   So velocity should be added here (it is mentioned in the sentence above).